### PR TITLE
Update Orders.cshtml

### PR DESCRIPTION
**Pull request by nlandegren**

### CODE REVIEW

**Review by philiphenningsson — CHANGES_REQUESTED** [review]
looking good my friend

File: HakimLivs/Pages/Admin/Orders.cshtml [hunk 1]
```diff
@@ -51,7 +51,7 @@
 									@Html.DisplayFor(c => order.Status)
 								</td>
 								<td>
-									<a class="btn btn-primary" asp-page="../Orders/Details" asp-route-id="@order.ID">Redigera</a>
+									<a class="btn btn-primary" asp-page="/Orders/Details" asp-route-id="@order.ID">Redigera</a>
 								</td>
 								<td>
 									<a class="btn btn-primary" asp-page="/Orders/Delete" asp-route-id="@order.ID">Ta bort</a>
```
Comment (philiphenningsson) [inline HakimLivs/Pages/Admin/Orders.cshtml:57]: very nice my friend

Comment (nlandegren) [reply HakimLivs/Pages/Admin/Orders.cshtml:57]: thx

Comment (nlandegren) [reply HakimLivs/Pages/Admin/Orders.cshtml:57]: ty

**Review by philiphenningsson — APPROVED** [review]

**Review by nlandegren — COMMENTED** [review]
reviewed ur review, nice

**Pull request merged**